tests/test_image_seq2seq.py

```diff
@@ -142,6 +149,43 @@ def test_multitask_early_fusion(self):
             valid['ppl'], 5.0, 'failed to train image_seq2seq on image+text task'
         )
 
+    @testing_utils.retry(ntries=3)
+    @testing_utils.skipUnlessGPU
+    def test_image_task_spatial_features(self):
+        """
+        Test that model correctly handles image task.
+
+        No training, only eval
```

Comment:
Hmm - wouldn't it be training because you're calling `testing_utils.train_model()`?

Reply:
ha yes, actually need to change the docstring in an above test as well